lib/http_server_handler/routes.ex

```diff
@@ -0,0 +1,7 @@
+defmodule HTTPServer.Routes do
+  alias HTTPServer.Request
+  # alias HTTPServer.Handler
```

Comment:
nit: remove commented code if no longer needed 😄

Reply:
Great catch! This was also being flagged as a problem (even though it wasn't being used?) so I'm glad you pointed it out!